samples/XCT.Sample/ViewModels/AboutViewModel.cs

```diff
@@ -12,9 +12,9 @@ public class AboutViewModel : BaseViewModel
 	{
 		readonly GitHubClient gitHubClient = new GitHubClient(new ProductHeaderValue("XamarinCommunityToolkitSample"));
 
-		RepositoryContributor[] contributors = new RepositoryContributor[0];
+		RepositoryContributor[] contributors = Enumerable.Empty<RepositoryContributor>().ToArray();
```

Comment:
`Array.Empty<RepositoryContributor>()`

Reply:
Can't - .NET Standard 1.0 doesn't support `Array.Empty` https://docs.microsoft.com/en-us/dotnet/api/system.array.empty?view=net-5.0

Reply:
But this is Sample app. It targets `netstandard2.0`

Reply:
Oh, good catch!